app/views/comments/_form.html.erb

```diff
@@ -69,11 +69,16 @@
   <script>
 
   (function() {
+    $('#text-input').bind('keypress',function(e){
+      if (e.ctrlKey && e.keyCode == 10) {
+        $("#comment-form .btn-primary").click();
+      }
+    })
+  
     $('#comment-form').bind('ajax:beforeSend', function(event){
       $("#text-input").prop('disabled',true)
       $("#comment-form .btn-primary").button('loading',true);
     });
```

Comment:
@sk9331657  please add a newline after this line, you have removed 3 lines
thanks :)

Reply:
@avsingh999  Now is it ok??